.github/workflows/reusable-doc-test.yaml

```diff
@@ -17,92 +17,47 @@ on:
 #   workflow_dispatch:
 
 env:
-  # REQUIRED organization level variables used in this workflow.
-  #   ${{ github.event.repository.default_branch }} is used throughout
-  #     rather than main to generalize default branch
-  #     (removed DEFAULT_BRANCH org var)
-  #   DOCKER_REGISTRY: URL to the docker registry
-  #   DOCKER_REGISTRY_USER: user to sign into DOCKER_REGISTRY
-  #   DOCKER_REGISTRY_TOKEN: token for USER user to sign into DOCKER_REGISTRY
-  #   RUNNER: default runner
-  #   UPLOAD_GITHUB_ARTIFACTS: true/false, specify whether to upload artifacts to github
-  #   USE_DOCKER_BUILDX_ACTION: true/false, specify whether to use the docker buildx action
-
-  # OPTIONAL organization level variables:
-  #   # Some system implementations may require different runners for building and
-  #   # running docker containers - allow specifying separately. Defaults to vars.RUNNER
-  #   # if these are not defined. Note env context nor strings are supported in the
-  #   # runs-on field, so we MUST use the vars context for the default.
-  #   RUNNER_DOCKERBUILD: runner to build docker containers
-  #   RUNNER_DOCKERRUN: runner to run docker containers
-
   # The path to the GeoIPS package in DOCLINTTEST images
   BASE_GEOIPS_PATH: /packages/geoips
-  # Allow overriding the doclinttest image tag used for plugin packages
-  # Allows using a different version of the doc, lint, and test code
-  PLUGIN_DOCLINTTEST_TAG: doclinttest-stable
 
 defaults:
   run:
     shell: bash
 
 jobs:
+  # Build the base GeoIPS doclinttest image
+  # This image is used in all subsequent tests
+  # It is only built for the main GeoIPS package but is used by all repositories
+  #
+  # Within this image, the GeoIPS package is located in /packages/geoips and is
+  # installed using `pip install .[doc,lint,test]`. Scripts from geoips can be called
+  # from /packages/geoips.
+  get-doclinttest-image:
+    name: Get docker image
+    # You do not appear to be able to use variables in this "uses" field.
+    uses: NRLMMD-GEOIPS/geoips_ci/.github/workflows/reusable-build-docker-image.yaml@build_stable_image_on_tag
+    secrets:
+      token: ${{ secrets.token }}
+      docker_token: ${{ secrets.docker_token }}
+    permissions:
+      packages: write
+      contents: write
+
```

Comment:
This was all that was needed to call the reusable build docker image workflow.  Note we had to add "outputs" to the reusable workflow, so we could reference the DOCKER_IMAGE_NAME here.

And it still works, see data_fusion repository (successful doc-test.yaml):

https://github.com/NRLMMD-GEOIPS/data_fusion/actions/runs/12698134825?pr=39

![image](https://github.com/user-attachments/assets/b2995b90-21be-4a96-a5a0-c5f6ab400fcc)